account/urls.py

```diff
@@ -13,9 +13,11 @@
 
 # local imports.
 from account.views import Punch
+from account.views  import GetStartTime
 
 app_name = 'accounts'
 
 urlpatterns = [
     path('punch', Punch.as_view(), name='punch in punch out'), 
+    path('GetStartTime', GetStartTime.as_view(), name = "get start time")
```

Comment:
this should be the name of the API, check your issue for API description and make sure the pathname here matches that description.